frontend/www/js/omegaup/components/certificate/Details.vue

```diff
@@ -7,9 +7,7 @@
     </div>
     <div class="panel-body">
       <span
-        v-html="
-          ui.formatString(T.certificateDetailsBody, { uuid: this.uuid })
-        "
+        v-html="ui.formatString(T.certificateDetailsBody, { uuid: this.uuid })"
```

Comment:
Ya sólo faltaría esto: 
```suggestion
        v-html="ui.formatString(T.certificateDetailsBody, { uuid: urlencode(uuid) })"
```